server/server.go

```diff
@@ -283,6 +287,13 @@ func NewServer(config *Config) (*Server, error) {
 		return nil, err
 	}
 
+	// start relayer
+	if config.Relayer {
+		if err := m.setupRelayer(); err != nil {
```

Comment:
Why is the relayer part of server? The relayer is an entity that in the current scope only lives inside `polybft`.

Reply:
FTR: IMO we need the flag in server, so why don't have it properly confined to server instead of burying it in polybft?

Reply:
I'm leaning more towards Ferran's opinion, although it is not really black vs white situation here:
Relayer is polybft specific and is coupled with bridge workflow. So I'd say we should instantiate it in the polybft in case Bridge is enabled (although we can agree if we want to have additional relayer flag separately configured independently if validator is running in bridge mode or not). 
The only challenge I see currently is are we able to easily retrieve all the server configuration required by the relayer in the polybft itself (like data dir, json rpc address etc.)

Reply:
Agreement FTR https://github.com/0xPolygon/polygon-edge/pull/1044#issuecomment-1363895539